spec/requests/blood_oxygen_conditions_spec.rb

```diff
@@ -0,0 +1,109 @@
+require 'rails_helper'
+
+RSpec.describe 'BloodOxygenConditionsController', type: :request do
```

Comment:
Saran dari mentor kemarin, nice to have kalo spec/request dipindah ke folder spec/controller dan namanya ngikut nama file controller-nya dikasih suffix _spec. Biar seragam semua penamaan folder/file nya